### PR TITLE
fix plugin request

### PR DESCRIPTION
**Pull request by lilyanB**
fix the name of the wallet plugin
resolve post request to run the install

### CODE REVIEW

File: web/massastation/src/pages/Index/Index.tsx [hunk 1]
```diff
@@ -39,18 +39,17 @@ export function Index() {
   const { data: availablePlugins } =
     useResource<PluginStoreItemRequest[]>('plugin-store');
 
-  const { mutate, isSuccess, isError } = usePost<any, any>(
-    `plugin-manager?source=${urlPlugin}`,
-  );
+  const { mutate, isSuccess, isError } = usePost<any, any>(`plugin-manager`);
```
Comment (Pierregvx) [inline web/massastation/src/pages/Index/Index.tsx:42]: can you specify the type ?

**Review by Pierregvx — CHANGES_REQUESTED** [review]

File: web/massastation/src/pages/Index/Index.tsx (hunk 1 above)
Comment (Thykof) [inline web/massastation/src/pages/Index/Index.tsx:42]:
```suggestion
  const { mutate, isSuccess, isError } = usePost<any, any>('plugin-manager');
```

**Review by Thykof — CHANGES_REQUESTED** [review]

File: web/massastation/src/pages/Index/Index.tsx (hunk 1 above)
Comment (lilyanB) [reply web/massastation/src/pages/Index/Index.tsx]: done

**Review by Pierregvx — APPROVED** [review]

File: web/massastation/src/pages/Index/Index.tsx [hunk 2]
```diff
@@ -39,18 +39,17 @@ export function Index() {
   const { data: availablePlugins } =
     useResource<PluginStoreItemRequest[]>('plugin-store');
 
-  const { mutate, isSuccess, isError } = usePost<any, any>(
-    `plugin-manager?source=${urlPlugin}`,
-  );
+  const { mutate, isSuccess, isError } = usePost<any, any>('plugin-manager');
```
Comment (Thykof) [inline web/massastation/src/pages/Index/Index.tsx:42]:
```suggestion
  const { mutate, isSuccess, isError } = usePost<null>('plugin-manager');
```

File: web/massastation/src/pages/Index/Index.tsx (hunk 1 above)
Comment (Thykof) [reply web/massastation/src/pages/Index/Index.tsx]: good catch!

Comment (lilyanB) [reply web/massastation/src/pages/Index/Index.tsx]: I don't know the type. The request doesn't return anything except a status.

File: web/massastation/src/pages/Index/Index.tsx (hunk 2 above)
Comment (lilyanB) [reply web/massastation/src/pages/Index/Index.tsx]: done

**Review by Thykof — APPROVED** [review]

**Pull request merged**